{"id":583,"details":"\n  mÌ©decin\n  \u003ctable\u003e\n    \u003ctr\u003e\n      \u003ctd\u003eRegion\u003c/td\u003e\n\u003ctd\u003eProvence-Alpes-CÌ«te d'Azur\u003c/td\u003e\n    \u003c/tr\u003e\n    \u003ctr\u003e\n\u003ctd\u003eDepartment\u003c/td\u003e\n\u003ctd\u003eHautes-Alpes\u003c/td\u003e\n\u003c/tr\u003e\n    \u003ctr\u003e\n\u003ctd\u003eCity\u003c/td\u003e\n\u003ctd\u003eBernwiller \u003c/td\u003e\n\u003c/tr\u003e\n    \n  \u003c/table\u003e\n  ","patient":"Male Patient From Provence Alpes Cì«Te D'azur, Hautes Alpes, Bernwiller ","source":"unknown_source","status":"in_hospital","confirmed_at":"2020-03-07T00:00:00.000Z","created_at":"2020-03-16T10:12:56.767Z","updated_at":"2020-03-16T10:42:25.884Z","infected_by":"unknown_infection","number":309,"urls":["https://github.com/lperez31/coronavirus-france-dataset"],"recovered_at":null,"gender":"male","age":0,"nationality":"unidentified_nationality","symptomatic_at":null,"slug":"france-case-309-male-patient-from-provence-alpes-ci-te-d-azur-hautes-alpes-bernwiller","suggested_url":"france-case-309-male-patient-from-provence-alpes-ci-te-d-azur-hautes-alpes-bernwiller","hospital_stay_length_in_days":null,"days_from_symptoms_to_confirmation":null,"without_symptoms":false,"reporting_country":"fr"}